lib/shelley/src/Cardano/Wallet/Shelley/Launch.hs

```diff
@@ -316,6 +327,58 @@ parseGenesisData = \case
             , block0
             )
 
+-- | Returns the shelley test data path, which is usually relative to the
+-- package sources, but can be overridden by the @SHELLEY_TEST_DATA@ environment
+-- variable.
+getShelleyTestDataPath :: IO FilePath
+getShelleyTestDataPath = fromMaybe source <$> lookupEnvNonEmpty var
+  where
+    source = $(getTestData) </> "cardano-node-shelley"
+    var = "SHELLEY_TEST_DATA"
+
+lookupEnvNonEmpty :: String -> IO (Maybe String)
+lookupEnvNonEmpty = fmap nonEmpty . lookupEnv
+  where
+    nonEmpty (Just "") = Nothing
+    nonEmpty m = m
+
+minSeverityFromEnv :: Severity -> String -> IO Severity
+minSeverityFromEnv def var = lookupEnvNonEmpty var >>= \case
+    Nothing -> pure def
+    Just arg -> either die pure (parseLoggingSeverity arg)
+
+-- Allow configuring @cardano-node@ log level with the
+-- @CARDANO_NODE_TRACING_MIN_SEVERITY@ environment variable.
+nodeMinSeverityFromEnv :: IO Severity
+nodeMinSeverityFromEnv =
+    minSeverityFromEnv Info "CARDANO_NODE_TRACING_MIN_SEVERITY"
+
+-- Allow configuring integration tests and wallet log level with
+-- @CARDANO_WALLET_TRACING_MIN_SEVERITY@ environment variable.
+walletMinSeverityFromEnv :: IO Severity
+walletMinSeverityFromEnv =
+    minSeverityFromEnv Info "CARDANO_WALLET_TRACING_MIN_SEVERITY"
```

Comment:
This re-adds the verbose logs running `cardano-wallet:integration`

Reply:
Not sure how we want to fix

Reply:
A quick fix could be to let it take the default as an argument or return a `Maybe`.

Reply:
Woops. Probably a wrong merge resolution here :+1: